irc.markdown

```diff
@@ -6,11 +6,9 @@ isCommunity: true
 
 # IRC Channels
 
-The IRC channel can be an excellent place to learn more about Haskell, and to just keep in the loop on new things in the Haskell world. Many new developments in the Haskell world first appear on the IRC channel.
+The IRC channel can be an excellent place to learn more about Haskell, and to just keep in the loop on new things in the Haskell world. Many new developments in the Haskell world first appear on the IRC channel. Haskell channels have been moving from freenode and are now mainly established on libera.chat.
```

Comment:
```suggestion
The IRC channel can be an excellent place to learn more about Haskell, and to just keep in the loop on new things in the Haskell world. Many new developments in the Haskell world first appear on the IRC channel. Haskell channels have been moving from freenode and are now mainly established on [libera.chat](https://libera.chat).
```

May as well link to [libera.chat](libera.chat).

Reply:
Oops, looks like I messed up the link in the GitHub comment: [libera.chat](https://libera.chat) should work.